docs/connector-development/tutorials/build-a-connector-the-hard-way.md

```diff
@@ -1017,36 +969,26 @@ acceptance_tests:
 #      configured_catalog_path: "integration_tests/configured_catalog.json"
 #      future_state_path: "integration_tests/abnormal_state.json"
 ```
+To run the test suite, we'll use [`airbyte-ci`](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md).
```

Comment:
/cc @evantahler your PR took 5 minutes from my question on Slack, mine — closer to two weeks, but I got to cleaning it up. ;)